code/modules/mob/living/carbon/life.dm

```diff
@@ -590,11 +590,8 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 	if(!dna)
 		return
 	var/obj/item/organ/liver/liver = getorganslot(ORGAN_SLOT_LIVER)
-	if(liver)
-		if(liver.damage < liver.maxHealth)
-			return
-		liver.organ_flags |= ORGAN_FAILING

```

Comment:
`liver_failure` doesn't set this

Reply:
where should I handle it?

Reply:
I put it in liver_failure(), but you should know that the proc where it used to be was never called by anything before I touched it, and I don't think it actually does anything in liver_failure() (since liver_failure() will only be called if you don't have a liver or if the liver itself calls it (which would be after its parent organ on_life() proc ran, which would handle assigning the ORGAN_FAILING flag anyway).

But hey, whatever floats your boat.

Reply:
alright, I've put a check_damage_thresholds() check in liver_failure()

I'm not sure why you wanted me to add one, though, since this stuff should already be handled by applyorgandamage(). But hey, whatever floats your boat.